be/src/common/daemon.cpp

```diff
@@ -117,6 +119,53 @@ void Daemon::memory_maintenance_thread() {
     }
 }
 
+void Daemon::memory_proc_refresh_thread() {
+    int64_t interval_milliseconds = 500;
```

Comment:
change it to a config. If we find the refresh process has impact on normal workflow we could change this parameter to slow down refresh.

Reply:
done